.github/workflows/statesync.yml

```diff
@@ -0,0 +1,29 @@
+name: State Sync
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API.
+on: [push, pull_request, workflow_dispatch]
+
+# This workflow makes x86_64 binaries for mac, windows, and linux.
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: State Sync
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2.1.3
+        with: 
+          go-version: '^1.17'
+
+      - name: Compile 
+        run: go install ./...
+
+      - name: init
+        run: akash init test
+
+      - name: download genesis
+        run: wget -O ~/.akash/config/genesis.json https://cloudflare-ipfs.com/ipfs/QmZjxmPSrHw2zFN8HmDTY99SBT1C9meMchuj1bb43Tsway
```

Comment:
yep, that should work. 

I use ipfs for genesises, because after 3300 downloads we can trigger git lfs mode on github, but can change it for sure :)

Reply:
done